libs/designer/src/lib/core/utils/parameters/helper.ts

```diff
@@ -2306,7 +2306,7 @@ function swapInputsValueIfNeeded(inputsValue: any, manifest: OperationManifest)
  * Example: Batch trigger
  */
 function updateInputsValueForSpecialCases(inputsValue: any, allInputs: InputParameter[], customSwagger?: SwaggerParser) {
-  if (!inputsValue || typeof inputsValue !== 'object') {
+  if (!inputsValue || typeof inputsValue !== 'object' || Array.isArray(inputsValue)) {
```

Comment:
Why is the inputs Value getting changed here? It should only be executed or changed when we have property with serialization and deserialization set...it should not impact any other inputs..

Reply:
This isn't happening because of the serialization/deserialization, but rather because we're using clone() which clones into an object... The issue is this doesn't work with arrays

Reply:
I see, so can we fix it using copy instead of clone? Because i do not want to remove array from this equation when we support it.